### DISCUSSION TITLE
Small portion of the screen causes tablet cursor to leave the game window w/ dual monitor setup

### DISCUSSION THREAD

**Discussion by LargerManZ** [ppy/osu#18501] [Q&A] [answered]
### Type

Game behaviour

### Bug description

OS: Manjaro Linux

**I've been playing with two monitors and a tablet** and found a small spot on the screen where if you move your cursor over it, the cursor will leave the osu window and tap on the other screen I use. This often times causing the game to tab out and pause _in the middle of gameplay._

I did some testing with this and found that it doesn't work with mouse, and it happens regardless of if I'm using fullscreen or borderless windowed. In the main menu there is also spots that the cursor will point towards that are near this area. 

 I also believe that this spot is a moving target, or it's just bigger than I think and it doesn't happen 100% of the time. I'm not sure at all what's causing it. Might be caused by these error messages I get when starting up.

### Screenshots or videos

Startup Messages
![osu_2022-05-26_16-23-10](https://user-images.githubusercontent.com/62466706/171098425-3da8f345-4b15-4cdc-a179-a366cfee5f42.jpg)

### Version

Lazer 2022.523.0

### Logs

[database.log](https://github.com/ppy/osu/files/8802314/database.log)
[performance.log](https://github.com/ppy/osu/files/8802315/performance.log)
[network.log](https://github.com/ppy/osu/files/8802316/network.log)
[runtime.log](https://github.com/ppy/osu/files/8802317/runtime.log)

**Comment by peppy** [comment, marked as answer]
Sounds like you don't have in-game tablet handling enabled. Can you confirm?

**Reply by LargerManZ** [reply]
![osu_2022-05-30_22-43-03](https://user-images.githubusercontent.com/62466706/171101119-fd7e081d-7ee6-44a3-a9c0-9699d2458eca.jpg)
This is it right?

**Reply by LargerManZ** [reply]
I've had this on since the beginning.

**Reply by LargerManZ** [reply]
My problem is not solved.

**Reply by peppy** [reply]
Can you also use your tablet to move around on your linux desktop? If so, whatever is doing that movement may be clicking outside of the osu! window. Please disable whatever is doing that.

**Reply by LargerManZ** [reply]
You're probably right about that, I'm not sure what is allowing me to do that. Let me look into it.

**Reply by LargerManZ** [reply]
I cannot find what is allowing my tablet to work properly outside of lazer. I already properly uninstalled opentabletdriver a couple days ago

**Reply by LargerManZ** [reply]
wait I think I found it. let me try it out.

**Reply by LargerManZ** [reply]
I think my osu application's tablet stuff bricked when I installed opentabletdriver last. I will reinstall and see if that fixes it.

**Reply by LargerManZ** [reply]
Alright, I reinstalled and am having a different message now:

DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw3.

**Reply by smoogipoo** [reply]
Read the wiki: https://opentabletdriver.net/Wiki/FAQ/Linux#fail-device-streams

**Reply by LargerManZ** [reply]
I followed the instructions and now I'm back to where I was before with the argument is out of range exeption

**Reply by LargerManZ** [reply]
is it a compatibility issue with xppen star g430s?

**Reply by smoogipoo** [reply]
Possibly, I suggest you ask on their GitHub then: https://github.com/OpenTabletDriver/OpenTabletDriver
